zenodo/modules/spam/templates/zenodo_spam/email/spam_user_email.tpl

```diff
@@ -0,0 +1,27 @@
+{#
```

Comment:
I would simply put this in the flash message when they publish, and not send an email.

Reply:
The only issue that this is not persistent so then if a user clicks on another tab they will be locked out and if they haven't read the error message they have no awareness of what happened.